app/controllers/tax_center_controller.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+class TaxCenterController < Sellers::BaseController
+  layout "inertia", only: [:index]
+  before_action :ensure_tax_center_enabled
+
+  def index
+    authorize :balance
+
+    @title = "Payouts"
+    year = params[:year]&.to_i || Time.current.year
+
+    tax_center_presenter = TaxCenterPresenter.new(seller: current_seller, year:)
+
+    render inertia: "TaxCenter/Index",
+           props: { tax_center_presenter: tax_center_presenter.props }
+  end
+
+  def download
+    authorize :balance, :index?
+
+    year = params[:year].to_i
+    form_type = params[:form_type]
+
+    error_message = "Tax form not available for download."
+
+    tax_form = current_seller.user_tax_forms.for_year(year).where(tax_form_type: form_type).first
+    if tax_form.blank?
+      redirect_to tax_center_path(year:), alert: error_message
+      return
+    end
+
+    stripe_account_id = current_seller.stripe_account&.charge_processor_merchant_id
```

Comment:
It's possible that past tax forms for a user were filed with a different Stripe account, and currently there may be no active Stripe account or current Stripe account may be different. In which case we'll need to check for forms against old `current_seller.merchant_accounts.stripe` as well I think. It's an edge case, but we should probably handle it.